docs/upgrade.md

```diff
@@ -1,11 +1,10 @@
 ## Upgrading Kubernetes
 
-Upgrading Kubernetes is easy with kops.  The cluster spec contains a KubernetesVersion, so you
-can simply edit it with `kops edit`, and apply the updated configuration to your cluster.
+Upgrading Kubernetes is easy with kops.  The cluster spec contains a `KubernetesVersion`, so you can simply edit it with `kops edit`, and apply the updated configuration to your cluster.
```

Comment:
What would specifying a binary look like? And do you mean that only valid released k8s versions are supported here?